runtime/codert_vm/cnathelp.cpp

```diff
@@ -69,7 +69,7 @@ buildJITResolveFrameWithPC(J9VMThread *currentThread, UDATA flags, UDATA parmCou
 	currentThread->jitException = NULL;
 	resolveFrame->specialFrameFlags = flags;
 #if defined(J9SW_JIT_HELPERS_PASS_PARAMETERS_ON_STACK)
-	resolveFrame->parmCount = parmCount;
+	resolveFrame->parmCount = 0;
```

Comment:
Why this change?  Is it required by the /GS flag?  

Changing this indicates that the resolveFrame doesn't have any parameters, which means we won't attempt to walk them / update them / etc.

`J9SW_JIT_HELPERS_PASS_PARAMETERS_ON_STACK` is like a x86-32 thing.  Does /GS apply to both 32/64?

Reply:
The change was attempting to address this concern:
>Some of the JIT helpers (c code) run on the java stack now in c interpreter.  We'd need to keep the flag disabled for those helpers.  The file to disable the option for is cnathelp.cpp

/GS applies to all platforms according to its documentation.

Reply:
That should result in makefile changes to disable the /GS flag for those files.

Reply:
ah.. that makes more sense. thanks!

Reply:
Made changes to correct this in the last push.